src/IPub/FormDateTime/Controls/Date.php

```diff
@@ -458,6 +458,11 @@ public function setValue($value)
 			$rawValue = $value;
 			$value = Utils\DateTime::createFromFormat($this->getDateFormat(TRUE), $value);
 
+			// Try default format
+			if ($value === FALSE) {
+				$value = Utils\DateTime::createFromFormat(\DateTime::ISO8601, $value);
```

Comment:
Please use `ATOM` instead `ISO8601`, it was wrongly implemented in PHP, so they fixed in `ATOM` constant

Reply:
done